### PR TITLE
Update red wave

### PR DESCRIPTION
**Pull request by Rasmagoric**

### CODE REVIEW

File: CustomThemesPackage/Red Wave.xaml [hunk 1]
```diff
@@ -6,46 +6,74 @@
...
             
             <!-- Accent Color -->
-            <Color x:Key="SystemAccentColor">#eee</Color>
-            <Color x:Key="SystemAccentColorLight2">#eee</Color>
-
-            <CornerRadius x:Key="ControlCornerRadius">2</CornerRadius>
-            <CornerRadius x:Key="OverlayCornerRadius">2</CornerRadius>
+            <Color x:Key="SystemAccentColor">#FFF</Color>
+            <Color x:Key="SystemAccentColorLight2">#FFF</Color>
+            <Color x:Key="SystemAccentColorDark1">#FFF</Color>
```
Comment (yaichenbaum) [inline CustomThemesPackage/Red Wave.xaml:19]:
```suggestion
```

File: CustomThemesPackage/Red Wave.xaml [hunk 2]
```diff
@@ -6,46 +6,74 @@
...
+            
+            <!-- Corner Roundness -->
+            <CornerRadius x:Key="ControlCornerRadius">3</CornerRadius>
+            <CornerRadius x:Key="OverlayCornerRadius">3</CornerRadius>
```
Comment (yaichenbaum) [inline CustomThemesPackage/Red Wave.xaml:23]: Any reason behind this change?

File: CustomThemesPackage/Red Wave.xaml [hunk 3]
```diff
@@ -6,46 +6,74 @@
...
         <ResourceDictionary x:Key="Default">
 
-            <Color x:Key="ChromeHigh">#d45332</Color>
-
-            <Color x:Key="SolidBackground">#db3f18</Color>
-            <Color x:Key="SolidBackgroundFillColorBase">#db3f18</Color>
-            <Color x:Key="SolidBackgroundFillColorSecondary">#dd4220</Color>
-            <Color x:Key="SolidBackgroundFillColorTertiary">#d45332</Color>
-            <Color x:Key="SolidBackgroundFillColorQuarternary">#C9401D</Color>
-            <Color x:Key="SolidBackgroundAcrylic">#db3f18</Color>
```
Comment (yaichenbaum) [inline CustomThemesPackage/Red Wave.xaml:16]: Is it possible to keep the acrylic value for users on Windows 10?

File: CustomThemesPackage/Red Wave.xaml [hunk 4]
```diff
@@ -6,46 +6,74 @@
...
+                BackgroundSource="HostBackdrop"
+                FallbackColor="#CE3A15"
+                TintColor="#CE3A15"
+                TintOpacity="0.9" />
+            <AcrylicBrush
+                x:Key="ControlStrokeColorSecondary"
+                BackgroundSource="HostBackdrop"
+                FallbackColor="#CE3A15"
+                TintColor="#CE3A15"
+                TintOpacity="0.9" />
```
Comment (yaichenbaum) [inline CustomThemesPackage/Red Wave.xaml:37]: @winston-de is it possible to use acrylic for strokes?

File: CustomThemesPackage/Red Wave.xaml [hunk 5]
```diff
@@ -55,121 +83,128 @@
...
+            <AcrylicBrush
+                x:Key="SidebarContentBackgroundBrush"
+                BackgroundSource="HostBackdrop"
+                FallbackColor="{ThemeResource SolidBackgroundFillColorBase}"
+                TintColor="{ThemeResource SolidBackgroundFillColorBase}"
+                TintOpacity="0.98" />
+
+            <!-- Accent Color -->
+            <Color x:Key="SystemAccentColor">#111</Color>
+            <Color x:Key="SystemAccentColorLight2">#111</Color>
```
Comment (yaichenbaum) [inline CustomThemesPackage/Red Wave.xaml:162]:
```suggestion
```

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: CustomThemesPackage/Red Wave.xaml (hunk 4 above)
Comment (winston-de) [reply CustomThemesPackage/Red Wave.xaml:37]: In this instance, no. ControlStrokeColorSecondary and the others are expected to be of type ``Color``, making it a brush can cause a crash when a solid color brush attempts to bind it's Color property to it.

Comment (winston-de) [reply CustomThemesPackage/Red Wave.xaml:37]: Please switch to overriding the brushes themselves with acrylic brushes, eg change `ControlStrokeColorDefault` to `ControlStrokeColorDefaultBrush`

**Pull request closed without merging**